tests/open-ce-tests.yaml

```diff
@@ -3,9 +3,9 @@ tests:
     command: |
         conda install -y git
         git clone --recursive -b v$(python -c "import xgboost; print(xgboost.__version__)") https://github.com/dmlc/xgboost
-        conda install -y pytest mock hypothesis matplotlib pandas=1.5.* urllib3 psutil distributed scikit-learn=1.1.*
+        conda install -y pytest mock hypothesis matplotlib pandas=1.5.* urllib3 psutil distributed scikit-learn=1.3.*
         PYTHON_VER=$(python -c 'import platform; print(platform.python_version())')
-        if [[ ! "$PYTHON_VER" =~ "3.9" ]]
+        if [[ "$PYTHON_VER" =~ "3.9" ]]
```

Comment:
Can we remove is py39 section as we are now supporting py310 and py311.

Reply:
sure